tmt/__init__.py

```diff
@@ -1,8 +1,8 @@
 """ Test Management Tool """
 
-# starting with setuptools_scm 7.0 it's possible to import __version__ directly
-# to support setuptools_scm 6.0 (in EL9), we can only import version
-from tmt._version import version as __version__  # noqa: F401
+from importlib.metadata import version
```

Comment:
Maybe nitpicking, but, would it make sense to not import the name into this namespace, and instead call it via its full name, i.e. `importlib.metadata.version`? With `import ... from ...` we'd get both `tmt.version` and `tmt.__version__` which might be puzzling.

Reply:
Sounds good to me. Am afk for the the rest of weekend though.

Reply:
No worries, I didn't expect any reaction, you're not supposed to be online right now anyway ;)

Reply:
Changed as per suggestion. Thanks!